cleanup-db-python.py

```
@@ -0,0 +1,123 @@
+# Step 2: Copy database
+print("Step 2: Copying database from container...")
+subprocess.run(["docker", "cp",
+               "whk-services-ignition-1:/usr/local/bin/ignition/data/db/config.idb",
+               "/tmp/config-cleanup.idb"], check=True)
+print("✓ Database copied to /tmp/config-cleanup.idb")
+
+print()
+
+# Step 3: Run H2 SQL to delete records
+print("Step 3: Executing SQL to delete corrupted records...")
+h2_jar = "/tmp/h2.jar"
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/h2.jar"

(S108)